tests/manager/fixtures/example-resource-presets.json

```diff
@@ -1,29 +1,52 @@
 {
   "resource_presets": [
     {
-      "name": "01-small",
+      "name": "cpu01-small",
+      "resource_slots": {
+        "cpu": "1",
+        "mem": "1073741824"
+      }
+    },
+    {
+      "name": "cpu02-medium",
+      "resource_slots": {
+        "cpu": "4",
+        "mem": "8589934592"
+      },
+      "shared_memory": "1073741824"
+    },
+    {
+      "name": "cpu03-large",
```

Comment:
Why did resource-preset fixture change in this PR?

Reply:
think when this pr was first made I had old preset but when I merged main branch in this pr `example-resource-presents.json` has changed.
this change 
my pr 
old ---- merged   -----> new
main      |
old  --- new

https://github.com/lablup/backend.ai/commit/8aaf27850ae87cec7572d95a0fe470f4b308f52b
between time when I worked on this branch and now there is pr from Joongi Kim which changed this fixture